External/Plugins/HaXeContext/Context.cs

```diff
@@ -819,7 +800,7 @@ public override MemberList ResolveImports(FileModel inFile)
                 foreach (ClassModel aClass in inFile.Classes)
                     if (aClass.Access != Visibility.Private) ResolveImport(aClass, imports);
             }
-
+            imports.Add(ResolveDefaults(inFile.Package));
```

Comment:
`import.hx` is "recursive".. e.g. if you have a file structure like this:

```
- src
    - import.hx
    - foo
        - import.hx
        - Bar.hx 
```

Then `Bar.hx` will have the imports of both `import.hx` files available. It doesn't look like this is covered by that? Also, it's not package, but directory-based (a file with the same package, but in another directory as an `import.hx`, will _not_ be affected by it).